### DISCUSSION TITLE
Multiple Users

### DISCUSSION THREAD

**Discussion by alexhogan18** [opengisch/QField#2790] [Q&A]
When using Qfield Cloud is it possible to have multiple users in the field at the same time collecting data, at the moment we have noticed that the when both users are collecting data and then uploading they are just over riding each others saved files meaning that data is being erased

**Comment by suricactus** [comment]
Can you give more details, what is the situation when the data gets erased in your opinition? All changes are collected and the data is not lost.

If they are modifying the same attributes on the same feature, then you might have a situation when the last one wins, depending on the setting "Overwrite conflicts".
![image](https://user-images.githubusercontent.com/2820439/164465305-a125f02b-fc0d-4599-98cc-2c2247e0a3fa.png)

**Reply by DefaultUser2** [reply]
I am looking to find this exact same option (Overwrite conflicts) in my own project but I cant find it. I can see that this seems to be some older UI - where can I find this option now?

**Reply by suricactus** [reply]
It is in the very same location as before. Project -> Settings

Please reach out the official support if you still experience problems.

**Comment by alexhogan18** [comment]
I think its more of an issue with the app, when pushing changes and syncing data it is not viewable on the other device and also some data is disappearing on the app following a sync, however the data was added to the project when viewing on qgis

**Comment by suricactus** [comment]
I marked the conversation as answered due to lack of details what are the steps to reproduce the issue.

QFieldCloud does not automatically sync the date on any of device.
